docs/books/admin_guide/03-commands.md

```diff
@@ -199,7 +199,7 @@ man 5 passwd
 
 will inform him about the files related to the command.
 
-Navigate through the manual with the arrows <kbd>↑</kbd> and <kbd>↓</kbd>. Exit the manual by pressing the <kbd>q</kbd> key.
+Navigate through the manual with the arrows `↑`and `↓`. Exit the manual by pressing the `q` key.
```

Comment:
@nishaaaaaant, for keyboard keys, we use this way: `++↑++`
See:
* https://squidfunk.github.io/mkdocs-material/reference/formatting/#adding-keyboard-keys
* https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#keys
* https://facelessuser.github.io/pymdown-extensions/extensions/keys/

I think, in that case, we can replace that key per:

```suggestion
Navigate through the manual with the arrows ++up++ and ++down++. Exit the manual by pressing the ++q++ key.
```

Reply:
I know it's a hard job we have to deal with

Reply:
@nishaaaaaant This document is a good reference for formatting, including the latest keyboard commands that @alemorvan indicated: https://docs.rockylinux.org/guides/contribute/rockydocs_formatting/#keyboard